_episodes/04-pytest.md

```diff
@@ -8,6 +8,9 @@ questions:
   - "I am a Fortran developer, do I need to care about pytest?"
 objectives:
   - "Get comfortable with pytest."
+keypoints:
+  - "pytest collects and runs all tests starting with `Test` or `test_`"
```

Comment:
For functions the prefix is `test_`. For classes it's `Test`. This can also be changed: http://doc.pytest.org/en/latest/example/pythoncollection.html#changing-naming-conventions
But from the above text one could think that functions can start with `Test` but these will be never run.

Reply:
thanks, good catch. I've now removed the mention of Test from the keypoint since it's not covered in the lesson, and we don't need to teach everything about pytest